packages/web-frontend/cypress/config/orgUnitMap.json

```diff
@@ -0,0 +1,175 @@
+{
+  "covidau": {
```

Comment:
_**Note:** the notes below apply only to configuration generated automatically using the `generateConfig.js` script_

A report may exist in multiple dashboard groups, which in turn may be defined for different organisation unit codes/levels and projects. Consider the following example:

1. We want to test the report `26`
2. This report exists under multiple dashboard groups, each one using a different org unit code: `CK`, `DL`, `FJ` etc. For each of those codes, the group is also available in multiple org unit levels: `country`, `district`, `sub_district` etc
3. Finally, each of those groups is defined in multiple projects `explore`, `disaster`, `imms` etc

❌One way around that would be submitting test data for each org unit that possibly meets a dashboard group's criteria. This way we will be sure that no matter which one is selected,  the rerport won't be empty. We are talking about thousands of org units 😱
✔Another way would be defining our preference for each possible combination of the above parameters. This is where `orgUnitMap.json` comes into play

For example, by defining
```json
{
  "TO": {
    "country": "TO",
    "district": "TO_Eua",
    "facility": "TO_CPMS",
    "village": "TO_Eua_Angaha"
  }
```

we know that if a dashboard group is selected that has `{ organisationUnitCode: 'TO', level: 'facility' }`, the report will be tested under `TO_CPMS`. Thus, submitting data for a single Tongan facility would be enough for testing groups with those specifications.

Our config generation script will select the first dashboard group whose specs meet an org unit map entry. If it cannot find any, the report will be skipped.

Reply:
Another way around this problem would be changing the dashboard groups in the e2e test server to use DemoLand (or similar), where we just define a single/ a few org units for each level. But I intuitively think that for e2e tests we shouldn't interfere much with how the portal is actually set up.

Reply:
Finally, we may want to adjust the specific org unit map we use to make our test data submission even leaner (remove unused countries, specify priority of a country over another), but you get the idea.

Reply:
We could also add a generation script for this config file :-)

Reply:
Definitely! I actually used a script locally (and made a few tweaks). I then thought we may want to set the org unit map "in stone", so that our testers now exactly which org units to use for submitting test data.

I suggest adjusting this behaviour after having a clearer idea of how test data will be submitted in practice, what do you think? We may even allow multiple automatically generated configs (random org unit, exhaustive report testing, etc)

Reply:
Yep I agree 👍